tools/circle_plus_gen/README.md

```diff
@@ -0,0 +1,74 @@
+# Circle+ generator
+
+Circle+ is a circle file which contains training hyperparameters. <br/> 
+This tool generates a circle+ by adding training hyperparameters to a circle file.<br/>
+It also helps handle circle+ file, such as checking whether the circle file contains training hyperparameters. <br/> 
+
+## Requirements
+
+This tool tested on python3.8. 
+
+1. (optional) Set python virtaul environment.
+
+    ```
+    python3 -m venv venv
+    source ./venv/bin/activate
+    ```
+
+2. Install required pakcages. 
+
+    Currently, only `flatbuffers==24.3.25` is needed.
```

Comment:
```suggestion
    Currently, only `flatbuffers` is needed.
```

I think this tool does not require a specific flatbuffers version, doesn't it?

Reply:
> I think this tool does not require a specific flatbuffers version, doesn't it?

Maybe. 
But I couldn't guarantee that it works well with other versions. 

So, How about
  - removing the version in README.md (because we don't have to emphasize the version in readme) 
  - keeping the version in requirements.txt  (because specifying the working well version is necessary)